### DISCUSSION TITLE
the student verification process

### DISCUSSION THREAD

**Discussion by zkllong-21** [community/community#61747] [GitHub Education] [answered]
### Select Topic Area

Question

### Body

Hi! My copilot is not activated on my [account settings](https://github.com/settings/copilot). I already got accepted at github education as a student 2 days ago but the benefits dont show up, I couldn't find what is average time that I have to wait for the benefits to be avaible.

**Comment by Akash1134** [comment, marked as answer]
Hi @zkllong-21 ,

Please check out the [step-by-step guide](https://techcommunity.microsoft.com/t5/educator-developer-blog/step-by-step-setting-up-github-student-and-github-copilot-as-an/ba-p/3736279) to setup GitHub Copilot as a verified student. 

Verify your student status on GitHub: Go to the GitHub Student Developer Pack and verify your student status on GitHub: [https://education.github.com/pack](https://education.github.com/pack?WT.mc_id=academic-88217-leestott)

Note that it might take a couple of days for verification to complete and benefits to reflect on your account. 

Thanks!

**Comment by AnasPuthawala** [comment]
Hi,
My student verification has been pending since last 10 days. Does anyone has any idea on how to contact the right person to get an update on this? 
Some times I get this type of error message when I try upload my student ID again.
<img width="1187" alt="Screenshot 2024-01-17 at 7 08 36 PM" src="https://github.com/community/community/assets/156557036/aadfd5cc-aaf3-4cf7-b01f-fc25e1ca9f98">

**Reply by rayyanshabir** [reply]
\same is happening with me bro
can you help us @Akash1134

**Comment by JustinP2459** [comment]
Good Afternoon,

I requested to be verified with my student information.  I verified my student email and sent a document that has my class schedule with my name and the name of the University.  It says I need additional documentation to show that I am a virtual or remote learner, it says the courses are Hybrid on the document that was sent. 
![Screenshot_2-5-2024_125815_sah soar cci nu edu](https://github.com/community/community/assets/112779610/a956640d-684e-4766-b110-fcc080b84531)
